README.md

```diff
@@ -1 +1,156 @@
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 SCHOOL LIBRARY <a name="about-project"></a>
+
+
+**SCHOOL LIBRARY** is an app that allows a user to record what books are in the library and who borrows them. The app allows you to: 
+| Add new students or teachers.
+| Add new books.
+| Save records of who borrowed a given book and when.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd school-library
+  git clone git@github.com:kaizipaul/school-library.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+  ruby person.rb
+```
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Paul Kaizirege Vedasto**
+
+- GitHub: [GitHub](https://github.com/kaizipaul)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kaizipaul/)
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features to add to the project.
+
+- [ ] **[SET UP ASSOCIATIONS]**
+- [ ] **[ADD BASIC UI]**
+- [ ] **[REFACTOR CODE]**
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Do I have to have any previous knowledge about ruby to use this project?**
+
+  - No!
+
+- **How to use this project?**
+
+  - check the [Getting Started](#getting-started) section to have a complete guid on how to setup and run teh project.
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
**[Optional]**

- [ ] Kindly remove these instructions because these are just for you if you wanna make a license file in your repository 🤔 These instructions are not for the viewers of your repository and also these instructions are making your readme look not clean 😨